cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -1546,22 +1546,27 @@ if (testing)
   # FIXME: Remove our version of gtest in roottest. We can reuse this one.
   # Add googletest
   # http://stackoverflow.com/questions/9689183/cmake-googletest
+  if(${CMAKE_BUILD_TYPE} STREQUAL "Debug")
+    set(debug_name "d")
```

Comment:
But do we really want gtest in debug, just because ROOT is in debug? I.e. why not
```
                  -DCMAKE_BUILD_TYPE=Release
```
unconditionally in https://github.com/root-project/root/pull/5228/files#diff-2e4217e5e2625aa4956d5240a2acb92eR1589 ?